src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -1,10 +1,18 @@
 {% trans_default_domain "locationview" %}
 
 {% if draftVersions %}
+    {{ form_start(draftActionsForm, {"action": path("ez_hybrid_platform_ui_content_view_draft_actions"), 'attr': {'class': 'ez-js-standard-form'}}) }}
```

Comment:
yeah that's the whole point of the app :) (btw that's documented in https://github.com/ezsystems/hybrid-platform-ui/blob/master/docs/howto/prevent_navigation_enhancement.md)
As you wrote, if you don't put it, the form is submitted in AJAX which should not be an issue but maybe you experienced any issue with that ?